common/src/main/java/com/wynntils/services/mapdata/providers/builtin/PlayerProvider.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright © Wynntils 2023-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.mapdata.providers.builtin;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Services;
+import com.wynntils.features.map.MainMapFeature;
+import com.wynntils.features.map.MinimapFeature;
+import com.wynntils.services.hades.HadesUser;
+import com.wynntils.services.hades.event.HadesEvent;
+import com.wynntils.services.hades.event.HadesUserEvent;
+import com.wynntils.services.mapdata.attributes.AbstractMapAttributes;
+import com.wynntils.services.mapdata.attributes.FixedMapVisibility;
+import com.wynntils.services.mapdata.attributes.type.MapAttributes;
+import com.wynntils.services.mapdata.attributes.type.MapDecoration;
+import com.wynntils.services.mapdata.attributes.type.MapVisibility;
+import com.wynntils.services.mapdata.type.MapCategory;
+import com.wynntils.services.mapdata.type.MapFeature;
+import com.wynntils.services.mapdata.type.MapLocation;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.mc.SkinUtils;
+import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.render.buffered.BufferedRenderUtils;
+import com.wynntils.utils.render.type.HealthTexture;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.stream.Stream;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class PlayerProvider extends BuiltInProvider {
+    private static final List<RemotePlayerLocation> PLAYERS = new ArrayList<>();
+
+    // Provided features and categories
+    private static final List<MapCategory> PROVIDED_CATEGORIES = List.of(new PlayersCategory());
+
+    @SubscribeEvent
+    public void onHadesAuthenticatedEvent(HadesEvent.Authenticated event) {
```

Comment:
Same here, you can combine these into a single `HadesEvent` listener. (And of course with `HadesUserEvent` below, too.)